<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>totalizar</class>
 <widget class="QDialog" name="totalizar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Totalizaci√≥n</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Forma de pago</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="1">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Cambio</string>
       </property>
       <property name="buddy">
        <cstring>lineEditCambio</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditDescuento">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="maxLength">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLineEdit" name="lineEditImporte">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="lineEditTotal">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>E&amp;ntrega</string>
       </property>
       <property name="buddy">
        <cstring>lineEditEntrega</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="lineEditEntrega">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLineEdit" name="lineEditCambio">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Descuento</string>
       </property>
       <property name="buddy">
        <cstring>lineEditDescuento</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>I&amp;mporte</string>
       </property>
       <property name="buddy">
        <cstring>lineEditImporte</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Tota&amp;l</string>
       </property>
       <property name="buddy">
        <cstring>lineEditTotal</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButtonCobrar">
       <property name="text">
        <string>Cobrar</string>
       </property>
       <property name="shortcut">
        <string>F8</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonTicket">
       <property name="text">
        <string>Ticket</string>
       </property>
       <property name="shortcut">
        <string>F9</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonFactura">
       <property name="text">
        <string>Factura</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSalir">
       <property name="text">
        <string>Salir</string>
       </property>
       <property name="shortcut">
        <string>Esc</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditEntrega</tabstop>
  <tabstop>pushButtonCobrar</tabstop>
  <tabstop>pushButtonTicket</tabstop>
  <tabstop>pushButtonFactura</tabstop>
  <tabstop>pushButtonSalir</tabstop>
  <tabstop>lineEditCambio</tabstop>
  <tabstop>lineEditDescuento</tabstop>
  <tabstop>lineEditImporte</tabstop>
  <tabstop>lineEditTotal</tabstop>
  <tabstop>comboBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonSalir</sender>
   <signal>clicked()</signal>
   <receiver>totalizar</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCobrar</sender>
   <signal>clicked()</signal>
   <receiver>totalizar</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>59</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
